models/hh_moto_5ht.nestml

```diff
@@ -0,0 +1,165 @@
+/* BeginDocumentation
+Name: hh_moto_5ht_nestml - a motor neuron model in HH formalism with 5HT modulation.
+Repo: https://github.com/research-team/hh-moto-5ht
+
+Description:
+
+ hh_moto_5ht is an implementation of a spiking motor neuron using the Hodkin-Huxley
+ formalism according to the article by Booth V. Basically this model is an implementation
+ of the existing NEURON model by Moraud EM and Capogrosso M.
+
+ The parameter that represents 5HT modulation is `g_K_Ca_5ht`. When it equals `1.0`,
+ no modulation happens. An application of 5HT corresponds to its decrease. The default
+ value for it is `0.6`. This value was used in the Neuron simulator model. The range of
+ this parameter is `(0, 1]` but you are free to play with any value.
+
+ Post-synaptic currents and Spike Detection are the same as in hh_psc_alpha.
+
+References:
+
+ Muscle spindle feedback circuit by Moraud EM and Capogrosso M.
+ https://senselab.med.yale.edu/ModelDB/showmodel.cshtml?model=189786
+
+ Compartmental model of vertebrate motoneurons for Ca2+-dependent spiking and plateau potentials under pharmacological treatment.
+ Booth V1, Rinzel J, Kiehn O.
+ http://refhub.elsevier.com/S0896-6273(16)00010-6/sref4
+
+Sends: SpikeEvent
+
+Receives: SpikeEvent, CurrentEvent
+
+Authors: Aleksei Sanin (https://github.com/vogdb)
+SeeAlso: hh_psc_alpha
+
+*/
+neuron hh_moto_5ht:
+  state:
+    r integer # number of steps in the current refractory phase
+  end
+
+  initial_values:
+    V_m mV = -65. mV # Membrane potential
+	Ca_in mmol = Ca_in_init  # Inside Calcium concentration
+
```

Comment:
Indentation seems to be not consistent. Otherthan that the model is fine and after fixing indentation can added to the repository.